.env

```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://localhost:20217/cfh
```

Comment:
Please add this file to your gitignore, you should **NEVER** push sensitive files like this to git.